nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -1003,6 +1006,10 @@ def predict_step(self, batch, batch_idx=0, dataloader_idx=0, has_processed_signa
         text = [self.decoding.strip_special_tokens(t) for t in text]
         return text
 
+    @property
+    def adapter_module_names(self) -> List[str]:
+        return ['', 'encoder', 'transf_encoder', 'transf_decoder']
```

Comment:
reason for adding empty string in list ?

Reply:
Its default adapter if user just provides the name of the module but not a target (add_adapter('xyz', adapter_cfg)). In that case adapter name is '' and it usually defaults to `encoder` adapters